webui/react/package.json

```diff
@@ -82,7 +82,6 @@
     "@types/plotly.js": "^1.50.21",
     "@types/react": "^16.9.52",
     "@types/react-dom": "^16.9.8",
-    "@types/react-helmet": "^6.1.0",
```

Comment:
`@types/react-helmet-async` is not needed as it [provides its own definitions](https://www.npmjs.com/package/@types/react-helmet-async).